@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -80,29 +80,42 @@ describe("MapView", function() {
         }
     });
 
-    //
-    // This test is broken, because `setCameraGeolocationAndZoom` doesn't behave as expected, i.e
-    // it offsets actual `geoCenter` a litte.
-    //
-    // TODO: check who is right? this test or `setCameraGeolocationAndZoom` implementation
-    //
-    it.skip("Correctly sets geolocation and zoom", function() {
+    it("Correctly sets geolocation and zoom", function() {
```

Comment:
Please create dedicated tests for setting the geolocation and providing the geolocation in the constructor